### PR TITLE
Updating why section to help improve clarity.

### PR DESCRIPTION
**Pull request by Alicole**
Have grouped like bullet points together -- no longer 10 bullet points in one section.
Simplified language where it was very verbose.
Hopefully improved flow and clarity.

Possible other changes may help here, but this is a start.

### CODE REVIEW

File: practices/code-review.md [hunk 1]
```diff
@@ -11,21 +11,23 @@ While we can support "traditional" Code Reviews, we routinely review via **pull
...
-    * helps prevent knowledge silos.
-    * helps identify areas where code re-use may be of benefit ("should that be a component?").
-    * if the code requires so much domain-specifc knowledge it cannot be understood by other teams, maybe that code needs refactoring?
-    * generally aids onboarding and training.
-1. Ensure that any changes to front-end code conforms to the same standards and guidelines.
-1. Fosters discussion and collaboration.
-1. Fosters caution and code quality over temporary hacks.
+## Why do them?
+
+**PR reviews are cross-team helping to foster discussion and feedback**
```
Comment (jpw) [inline practices/code-review.md]: `cross-team, thereby helping` ?

File: practices/code-review.md [hunk 2]
```diff
@@ -11,21 +11,23 @@ While we can support "traditional" Code Reviews, we routinely review via **pull
...
+
+1. Knowledge sharing and transfer within and between teams:
+	* allows everyone to learn from their colleagues;
```
Comment (jpw) [inline practices/code-review.md]: not sure why they all have semi-colons on the end of the lines? not a pattern i am familiar with  🤔

File: practices/code-review.md (hunk 1 above)
Comment (jpw) [inline practices/code-review.md]: `cross-team, thereby helping` ?

File: practices/code-review.md (hunk 2 above)
Comment (Alicole) [reply practices/code-review.md]: Basically it's a grammatical pattern for grouping together related clauses that form a list.

File: practices/code-review.md (hunk 1 above)
Comment (Alicole) [reply practices/code-review.md]: Not sure what you mean here -- please clarify.

File: practices/code-review.md (hunk 2 above)
Comment (jpw) [reply practices/code-review.md]: but they are in a list already so already grouped? in any case, it goes against how we do all the other lists, so suggest just removing them please.

**Pull request merged**